config/common/group_vars/k8s-cluster/ck8s-k8s-cluster.yaml

```diff
@@ -0,0 +1,89 @@
+kube_apiserver_enable_admission_plugins:
+  - "PodSecurityPolicy"
+  - "NamespaceLifecycle"
+  - "LimitRanger"
+  - "ServiceAccount"
+  - "DefaultStorageClass"
+  - "DefaultTolerationSeconds"
+  - "MutatingAdmissionWebhook"
+  - "ValidatingAdmissionWebhook"
+  - "ResourceQuota"
+  - "NodeRestriction"
+
+kube_kubeadm_apiserver_extra_args:
+  delete-collection-workers: "3"
+
+kube_proxy_mode: iptables
+kube_proxy_metrics_bind_address: "0.0.0.0:10249"
+
+enable_nodelocaldns: false
+
+kubernetes_audit: true
+audit_log_path: "/var/log/kube-audit/kube-apiserver.log"
+audit_log_maxage: 7
+
```

Comment:
Should we set `audit_log_maxsize`? Default is 100mb
https://github.com/kubernetes-sigs/kubespray/blob/v2.14.2/roles/kubernetes/master/defaults/main/main.yml#L52
but if i'm not missremebering we generate 1gb+(?) of audit logs every day (i might be confusing this with apiserver logs, not sure).

Reply:
According to this https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/
maxsize is only related to rotation size

```text
The maximum size in megabytes of the audit log file before it gets rotated.
```

So I guess it's okey for it to rotate multiple times a day?

Reply:
Yep :)